packages/pharos/src/utils/scss/_mixins.scss

```diff
@@ -340,7 +340,7 @@
 /// @access public
 /// @group links
 @mixin link-base {
-  @include font-base($font-size: inherit, $line-height: inherit);
+  @include font-base($line-height: inherit);
```

Comment:
This fixes an issue that blows up when you try to run it through `cssnano`. I think it actually generates invalid css. We were passing `$font-size: inherit` to `font-base`. This resulted in a css rule like:

`letter-spacing: calc(inherit * -.02)`

What's more interesting is that we have a `letter-spacing` rule a few lines down in this file anyways. So it get's overriden either way.

Reply:
I'm wary of this change. I think it might be what's impacting the letter spacing we see in the diff of the Chromatic build.

Reply:
Ya that is peculiar. Will dig in more

Reply:
I'd think a conditional around the `letter-spacing` rule if `$font-size` is `inherit` might do the trick